src/dynamodb_encryption_sdk/internal/formatting/deserialize/attribute.py

```diff
@@ -220,17 +240,18 @@ def _deserialize_function(tag):
         :type tag: dynamodb_encryption_sdk.internal.identifiers.Tag
         :rtype: callable
         """
+        # for some reason pylint can't follow the Enum member attributes
         deserialize_functions = {
-            Tag.BINARY.tag: _deserialize_binary,
-            Tag.BINARY_SET.tag: _deserialize_binary_set,
-            Tag.NUMBER.tag: _deserialize_number,
-            Tag.NUMBER_SET.tag: _deserialize_number_set,
-            Tag.STRING.tag: _deserialize_string,
-            Tag.STRING_SET.tag: _deserialize_string_set,
-            Tag.BOOLEAN.tag: _deserialize_boolean,
-            Tag.NULL.tag: _deserialize_null,
-            Tag.LIST.tag: _deserialize_list,
-            Tag.MAP.tag: _deserialize_map,
+            Tag.BINARY.tag: _deserialize_binary,  # pylint: disable=no-member
```

Comment:
Should we just disable this check globally?

Reply:
No, it's a good check to have, it just seems to get confused with the way that we're constructing enums with custom attributes.